mitosheet/mitosheet/tests/step_performers/test_export_to_file.py

```diff
@@ -264,6 +283,115 @@ def test_transpiled_with_export_to_xlsx_format_rows_no_header():
 ])
 """
 
+
+CONDITIONAL_FORMATS = [
+    (
+        ['A'], 
+        [{'condition': 'greater', 'value': 5}], 
+        '#e72323', 
+        '#ffffff'
+    ),
+    (
+        ['A'],
+        [{'condition': 'greater', 'value': 4}],
+        '#abcdef', 
+        '#000000'
+    )
+]
+# This tests when the user exports a dataframe with row formatting without header formatting.
+@pytest.mark.parametrize("column_ids, filters, background_color, font_color", CONDITIONAL_FORMATS)
+def test_transpiled_with_export_to_xlsx_conditional_format(column_ids, filters, background_color, font_color):
+    df = pd.DataFrame({'A': [4, 5, 6]})
+    mito = create_mito_wrapper(df, arg_names=['df'])
+    mito.set_dataframe_format(0, {
+        'headers': {},
+        "columns": {},
+        "rows": {},
+        "border": {},
+        "conditional_formats": [
+            {
+                'format_uuid': '_hkyc4pcux',
+                'columnIDs': column_ids,
+                'filters': filters,
+                'color': font_color,
+                'backgroundColor': background_color
+            }
+        ]
+    })
+    filename = 'test_format_conditional.xlsx'
+    mito.export_to_file('excel', [0], filename)
+    assert "\n".join(mito.transpiled_code) == f"""from mitosheet.public.v3 import *
+import pandas as pd
+import numpy as np
+
+with pd.ExcelWriter(r\'test_format_conditional.xlsx\', engine="openpyxl") as writer:
+    df.to_excel(writer, sheet_name="df", index=False)
+    add_formatting_to_excel_sheet(writer, "df", df, 
+        conditional_formats=[
+            {{'columns': {column_ids}, 'filters': {filters}, 'font_color': '{font_color}', 'background_color': '{background_color}'}}
+        ]
+    )
+
+df_styler = df.style\\
+    .apply(lambda series: np.where(series > {filters[0]['value']}, 'color: {font_color}; background-color: {background_color}', None), subset={column_ids})
+"""
```

Comment:
Can we use the `get_cell_formatting` function to test that the correct cells have color in this test (and the other conditional formatting test as well)? I think adding the cell index to the parameterization, and then expecting it to be the colors you set for the background colors would be cool.

Reply:
Oh oops! Just added that, good catch